interface/main/messages/messages.php

```diff
@@ -255,6 +255,23 @@
        $form_message_status = 'New';
    }
     generate_form_field(array('data_type'=>1,'field_id'=>'message_status','list_id'=>'message_status','empty_title'=>'SKIP','order_by'=>'title'), $form_message_status); ?>
+    <?php
+	if($noteid){
+		$doc = sqlQuery('select document_id from pnotes where id = ?',array($noteid));
+		if(!empty($doc['document_id'])){
+			$d = new Document($doc['document_id']);
+			$enc_list = sqlStatement("SELECT fe.encounter,fe.date,openemr_postcalendar_categories.pc_catname FROM form_encounter AS fe ".
+        " left join openemr_postcalendar_categories on fe.pc_catid=openemr_postcalendar_categories.pc_catid  WHERE fe.pid = ? order by fe.date desc", array($prow['pid']));
+			$str_dob = "DOB:".$prow['DOB']." Age:".getPatientAge($prow['DOB']);
+			$pname = $prow['fname']." ".$prow['lname'];
+			echo xlt('Linked Image Report') . ":</b>\n";
+			echo "<a href='javascript:void(0);' "; 
+			echo "onClick=\"gotoReport(".$d->get_id().",'".$pname."',".$prow['pid'].",".$prow['pubpid'].",'".$str_dob."');\">";
```

Comment:
need to html each variable with attr(addslashes()). example:
``````
attr(addslashes($d->get_id()))